packages/core/src/api/FirmwareUpdateV3.ts

```
@@ -0,0 +1,376 @@
+          await Promise.race([typedCall('GetFeatures', 'Features', {}), timeoutPromise]);
+          clearTimeout(timeoutId);
+        } catch (error) {
+          clearTimeout(timeoutId);
+          throw error;
+        }
+      } catch (error) {
+        if (error.message.includes('Update mode')) {
+          const updateParts = error.message.split('Update mode ');
+          const progressValue = updateParts[1] ?? '0';
+          const progress = parseInt(progressValue, 10) || 0;
+          this.postProgressMessage(progress);
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)